### ISSUE TITLE
[E-Commerce] Activate e-commerce bundle sends error when using empty install profile 

### ISSUE THREAD

**Issue by Yivan** [pimcore/pimcore#2267]
With a fresh install of last stable Pimcore 5, in the backend go in extension and click install extension e-commerce.

This send back errors and break totally Pimcore 5.

```
The parameter "pimcore_ecommerce.decimal_scale" must be defined.

in appDevDebugProjectContainer.php (line 7697)
at appDevDebugProjectContainer->getParameter('pimcore_ecommerce.decimal_scale')in PimcoreEcommerceFrameworkBundle.php (line 83)
at PimcoreEcommerceFrameworkBundle->boot()in Kernel.php (line 143)
at Kernel->boot()in kernel.php (line 25)
at require_once('...\\pimcore\\config\\kernel.php')in app.php (line 50)
```

Aside question:
- I was looking a way to disable the plugin to be able to make Pimcore working again, but i don't find any information on this on the documentation.
Is there a way to disable manually a plugin when we cannot go in backend admin/console cli is broken, like changing by hand config files/database values ? If yes, maybe it could be documented as emergency solution ?

Thanks.

**Comment by fashxp** [comment]
have a look at extensions.php in /var/config - there you can activate / deactivate the bundles manually

**Comment by fashxp** [comment]
the problem is strange though, because for decimal_scale there should be set a default value.

**Comment by Yivan** [comment]
Tahnsk for your answer. I can add setup is on Windows (but we never had problem with Pimcore 4 with this setup). PHP7.1/MariaDB10.1.

If you need further information to track the problem I will be happy to provide them to you.

**Comment by fashxp** [comment]
windows is a very valuable information! 
other than that pimcore 5 runs on your setup?

**Comment by Yivan** [comment]
No we didn't had any other problem from now. Install was ok (if we didn't use a custom environment name like explained here: https://github.com/pimcore/pimcore/issues/2265, but those two issues are not related at all i think) and the cms demo works fine. We developped some custom widget too, and no problem.

**Maye I can add that we choose "Empty"  as install profil during the installation.**

To resume, it is reproduced like this, with an install on Windows (php7.1/mariadb10.1):
- download last stable ZIP from Pimcore website (so not from github)
- run install using the web install (so not using composer)
- choose profile "Empty"
- check all requirement (mainly green, some orange for optionnal ones - composer ok)
- install is ok
- connect to admin and go to extension and click install e-commerce... bug.

**Comment by boris93** [comment]
[e-commerce-error-invalidargumentexception](https://talk.pimcore.org/t/e-commerce-error-invalidargumentexception/82 )

**Comment by Yivan** [comment]
Thanks boris93.

@fashxp 
I managed to make it work by flushing the cache. The error disapear, and after that i can run manually `php "bin/console" assets:install web --env=prod --no-ansi --symlink --relative` from the cli.

I tried the process again from a fresh install (last stable), always on Windows (Pimcore 4&5 work great on this station), and the error is always here. As i click "Enable" it open an extjs window with error. I maybe the decimal error is throwed because of another previuous error in the request. Here is a screenshot, just after having clicked "Enable":

![error-ecommerce](https://user-images.githubusercontent.com/9025839/34425244-9d7ea124-ec2a-11e7-9c5c-9c5c61951d8a.png)

Feel free to ask me any precision @fashxp so this bug could be corrected (i know Windows is not officially supported, but this bug deserve to be corrected i think, as it will be a no-go for ecommerce on Windows because actually we cannot  enable the bundle without making some manual hacks).

Thanks.
